src/components/Pama/pama.css

```diff
@@ -8,10 +8,25 @@
   width: 100%;
 }
 
+.rating-box {
+  margin-top: 1em;
+  border-radius: 5px;
+  border: 1px solid #384E77;
+  padding: 5px;
+}
+
+.current-selection {
+  padding-left: 4px;
+}
+
 .half-view {
   width: 50%;
 }
 
+div.choice {
```

Comment:
We explicitly call out the other styles within pama.jsx.  Where does this come into play?

Reply:
I think it's stray; will prune.